apis/v1alpha1/httproute_types.go

```diff
@@ -119,7 +120,9 @@ type RouteTLSConfig struct {
 	// string for both the group and kind, the resource defaults to "secrets".
 	// An implementation may support other resources (for example, resource
 	// "mycertificates" in group "networking.acme.io").
+	//
 	// Support: Core (Kubernetes Secrets)
+	//
```

Comment:
It would be good to understand the best practices around whitespace/formatting here and try to document them wherever possible.

Reply:
This is ultimately markdown, so a newline is needed to separate the support tag into its own paragraph. Otherwise it runs on from the descriptive text.